CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 ## [1.14.0] - 2023-03-30
 
 ### Added
+- Add alternative labels national report inventory to common report format (#1558)
```

Comment:
This should be a change, as you've changed an existing class. And it is enough simply put instead
`- common reporting format (#1558)`
at the *end* of the changed section.